File: app/src/main/java/com.caique.everis.testeandroid/model/ClientData.java
package com.caique.everis.testeandroid.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class ClientData implements Serializable{

    @SerializedName("userId")
    private int userId;

    @SerializedName("name")
    private String name;

    @SerializedName("bankAccount")
    private String bankAccount;

    @SerializedName("agency")
    private String agency;

    @SerializedName("balance")
    private Double balance;


    public int getUserId() {
        return userId;
    }

    public void setUserId(int userId) {
        this.userId = userId;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getBankAccount() {
        return bankAccount;
    }

    public void setBankAccount(String bankAccount) {
        this.bankAccount = bankAccount;
    }

    public String getAgency() {
        return agency;
    }

    public void setAgency(String agency) {
        this.agency = agency;
    }

    public Double getBalance() {
        return balance;
    }

    public void setBalance(Double balance) {
        this.balance = balance;
    }
}

File: teste/app/src/main/java/com/example/ilealnod/SantanderProjectKotlin/Todos/Dependencias/ClientData.kt
package com.example.ilealnod.SantanderProjectKotlin.Todos.Dependencias

import com.google.gson.annotations.SerializedName

class ClientData {

    // Dependencia dos Dados do cliente

    @SerializedName("userId")
    var userId: Int? = 0
    @SerializedName("name")
    var name: String? = null
    @SerializedName("bankAccount")
    var bankAccount: String? = null
    @SerializedName("agency")
    var agency: String? = null
    @SerializedName("balance")
    var balance: Double? = 0.0

}
